src/wlapplication_mousewheel_options.cc

```diff
@@ -133,47 +133,46 @@ struct MousewheelHandlerOptions {
 
 static const std::map<MousewheelOptionID, MousewheelOption> mousewheel_options = {
    {MousewheelOptionID::kUIChangeValueInvertX,
-    MousewheelOption("change_value_x_invert", MousewheelOptionType::kBool, false)},
+    MousewheelOption("change_value_x_invert", MousewheelOptionType::kBool, 0u)},
```

Comment:
in this case using the correct value according to the vriable type decreases readability as we have a variable that can be both an Integer as representation of a defined keymod or a bool depending on the second value provided to the struct. 
to keep the indication that we are trying to provide a bool here (which is technically an integer that gets converted to a bool in the structs functions) would it be possible to cast false to integer or can we use some NOLINTS here to ignore these lines?